src/lambda/models/samLambdaRuntime.ts

```diff
@@ -17,34 +17,34 @@ export enum RuntimeFamily {
     Python,
     NodeJS,
     DotNetCore,
+    Go,
 }
 
 export type RuntimePackageType = 'Image' | 'Zip'
 
 // TODO: Consolidate all of the runtime constructs into a single <Runtime, Set<Runtime>> map
 //       We should be able to eliminate a fair amount of redundancy with that.
-export const nodeJsRuntimes: ImmutableSet<Runtime> = ImmutableSet<Runtime>([
```

Comment:
nit: Is this a case of formatting wars?

Reply:
I guess this is what my prettier thinks is best? 🤷‍♂️